### PR TITLE
community/ffmpeg: upgrade to 4.2

### PR DESCRIPTION
**Pull request by J0WI**
https://ffmpeg.org/security.html

### CODE REVIEW

**Comment by maxice8** [comment]
There are duplicates

**Comment by J0WI** [comment]
Fixed.

**Comment by maxice8** [comment]
There still are duplicate CVEs

**Comment by maxice8** [comment]
Second hunk has duplicates

```
#     - CVE-2018-1999010
#     - CVE-2018-1999011
#     - CVE-2018-1999012
#     - CVE-2018-1999013
#     - CVE-2018-1999014
#     - CVE-2018-1999015
```

File: community/ffmpeg/APKBUILD [hunk 1]
```diff
@@ -21,22 +21,35 @@ source="https://ffmpeg.org/releases/ffmpeg-$pkgver.tar.xz
...
+#     - CVE-2019-9721
+#     - CVE-2019-11338
+#     - CVE-2019-11339
+#   4.1.1-r0:
+#     - CVE-2019-1000016
+#   4.1-r0:
+#     - CVE-2018-13305
+#     - CVE-2018-15822
 #   4.0.2-r0:
-#     - CVE-2018-13300
```
Comment (maxice8) [inline community/ffmpeg/APKBUILD]: This CVE is just removed

Comment (maxice8) [reply community/ffmpeg/APKBUILD]: Nevermind saw it is there below

Comment (J0WI) [reply community/ffmpeg/APKBUILD]: This has been fixed in `3.4.3-r0`

**Comment by maxice8** [comment]
Merged

**Comment by J0WI** [comment]
Thanks!

**Pull request closed without merging**